source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Managed MTU for VM Networks
+category: feature
+authors: dholler
+feature_name: Managed MTU for VM Networks
+feature_modules: engine,provider-ovn
+feature_status: In Development
+---
+
+# Managed MTU for VM Networks
+
+## Summary
+
+VM networks, especially external logical [OVN networks][1], transports the
+Ethernet frames through a physical host network. The MTU of the VM network has
+to be configured small enough to respect the MTU of the physical network.
+The feature introduced here adds the ability to manage the MTU in a centralized
+way to assist the configuration of the MTU of each VM network.
+
+Feature progress is tracked on [Trello][14]. Related patches can be found on
+[gerrit topic:managed_mtu][15].
+
+
+## Owner
+
+*   Name: [Dominik Holler](https://github.com/dominikholler)
+
+*   Email: <dholler@redhat.com>
+
+
+## Detailed Description
+
+The logical network traffic of the VMs has to be transported by physical
+networks between the hosts. The Ethernet frames of the logical network are
+transported either
+* as plain Ethernet frames,
+* encapsulated in a VLAN or
+* encapsulated in [GENEVE][3] tunnels.
+
+### Plain Ethernet frames
+If the Ethernet frames of the logical network are transported as a plain Ethernet
+frames of the physical network, the MTU of the logical network must not be
+grater than the MTU of the physical network.
+
+### VLAN
+The same rule applies, if the Ethernet frames of the logical network is
+transported encapsulated in a VLAN, because the added VLAN header belongs to
```

Comment:
It that true though? With plain ethernet we just pass frames forward. With VLAN on the other hand, we use tagged ethernet frame to carry non-tagged logical network traffic. VLAN ID is in the header, but when used, it offsets the actual payload.

So does not that mean, that with VLAN encapsulation we are not limited by MTU of physical network, but the size of the payload?

Reply:
Sounds interesting, but I do not get it.
Isn't the MTU of physical network equal to the size of the payload on Ethernet layer?

Reply:
Yea, sorry, I think you are right. Payload size remains the same, size of the whole frame is increased with VLAN.